README.md

```diff
@@ -12,3 +12,31 @@ for the structure of an attribute.
 ### How are attributes created?
 An initial list of attributes needed by Hypertrace are seeded from `helm/configs` but they
 can also be dynamically registered and queried using the APIs of AttributeService.
+
+## How Hypertrace uses Attribute service?
+Below is the high level schematic diagram(of the push based model) of different components involved in attribute service:
+
+| ![space-1.jpg](https://imagizer.imageshack.com/v2/xq90/924/xN2Top.png) | 
```

Comment:
These images are slightly outdated. Needs to be fixed before we add them here.

Reply:
ohh! Who can you help me to get those updated @avinashkolluru ?

Reply:
I will remove those for now and we can update them and then add in README.